identity-composer-directive.js

```diff
@@ -49,6 +49,18 @@ function brIdentityComposer($rootScope, $window, brCredentialLibraryService) {
     model.allCredentials = [];
 
     model.claimsPartiallyFulfillable = false;
+    model.publicAccess = {
+      requested: false,
+      acknowledged: false
+    };
+
+    // TODO: Consider changing "cred:requestPublicAccess" to
+    // "cred:requestPersistentAccess" with a value of "publicAccess" so we can
+    // support consumers/inspectors providing their ID as a value as well
+    if(jsonld.hasProperty(model.consumerQuery, 'cred:requestPublicAccess')) {
+      model.publicAccess.requested = true;
+      delete(model.consumerQuery['cred:requestPublicAccess']);
```

Comment:
delete isn't a function, it's a keyword, so syntax is: `delete x;`. Also, do we need to do this deletion at all?